custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json

```diff
@@ -831,6 +831,23 @@
         },
         "type": "boolean"
       },
+      {
```

Comment:
can you add this to `ccs_record_cases.json` instead. we are trying to phase out this ucr